python/src/pyarrow/parquet.cc

```diff
@@ -0,0 +1,30 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This file is an artifact of former C++ code in the python module. Will remove this.